lib/types/src/module.rs

```diff
@@ -504,6 +504,12 @@ impl ModuleInfo {
         index.index() < self.num_imported_tables
     }
 
+    /// Test whether the given table index is growable. Will return `None` if the
+    /// index is out of bounds.
+    pub fn is_table_growable(&self, index: TableIndex) -> Option<bool> {
+        Some(self.tables.get(index)?.maximum.is_none())
+    }
+
```

Comment:
Move this logic out of the types into an utility function inside of llvm intrinsics, we need to have an API on the module very similar to the Web API

Reply:
Also, the function will likely be not used as we will replace the logic for checking if a table is static (fixed size) or dynamic (growable)